.github/workflows/docs.yaml

```diff
@@ -0,0 +1,24 @@
+---
+    name: Update-docs
+    on:               # yamllint disable-line rule:truthy
+      push:
+        branches:
+          - main
+    jobs:
+      transfer-file:
+        runs-on: ubuntu-latest
+        steps:
+          - name: Checkout
+            uses: actions/checkout@v3
+          - name: Pushes Readme file
+            uses: dmnemec/copy_file_to_another_repo_action@main
+            env:
+              GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+              API_TOKEN_GITHUB: ${{ secrets.ARC_DOCS_API_TOKEN_GITHUB }}
+            with:
+              source_file: "README.md"
+              destination_repo: "sourcefuse/arc-docs"
+              destination_folder: "docs/arc-iac-docs/modules/terraform-aws-refarch-waf"
```

Comment:
change destination folder name to
`docs/arc-iac-docs/modules/terraform-postgresql-aws-arc-mgmt`

Reply:
done